src/main/java/com/ashkan/moviesapi/controllers/MovieController.java

```diff
@@ -0,0 +1,47 @@
+package com.ashkan.moviesapi.controllers;
+
+import com.ashkan.moviesapi.entities.Movie;
+import com.ashkan.moviesapi.services.interfaces.MovieService;
+import com.fasterxml.jackson.databind.ser.FilterProvider;
+import com.fasterxml.jackson.databind.ser.impl.SimpleBeanPropertyFilter;
+import com.fasterxml.jackson.databind.ser.impl.SimpleFilterProvider;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.converter.json.MappingJacksonValue;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/movies")
+public class MovieController {
+
+    private MovieService movieService;
+    private SimpleBeanPropertyFilter filter;
+
+    @Autowired
+    public MovieController(MovieService movieService,ModelMapper modelMapper) {
```

Comment:
You have here this modelMapper that you could use to Map the Business Logic Objects with Entities or viceversa when you need, please review how to do that, this will help you a lot to separate that 2 layers.